services/cloudformation/src/main/resources/codegen-resources/customization.config

```diff
@@ -20,5 +20,6 @@
         "describeStackResources",
         "describeStackEvents",
         "getTemplateSummary"
-    ]
+    ],
+    "marshallNonAutoConstructedEmptyListsForQuery": true
```

Comment:
I wonder if this should just be the default behavior? Could you maybe sync up with Trevor on what the expected behavior for empty lists in Query is? 

Also if we end up keeping this can you add a comment referencing https://github.com/aws/aws-sdk-java/issues/721 so we know why this is needed?

Reply:
Will do

Reply:
Spoke with Trevor. Absent and empty lists are semantically different in query land, so this looks like it's fine to do as a default.